### DISCUSSION TITLE
Recently viewed, not showing mcaps opened via finder

### DISCUSSION THREAD

**Discussion by mrkbac** [foxglove/community#996] [Bugs]
MCAPs open via finder (double clicked) are not logged in the "Recently viewed" list.

**Comment by defunctzombie** [comment]
Internal tracking issue: https://linear.app/foxglove/issue/FG-7480/double-clicked-mcap-file-on-macos-not-appearing-in-recents

**Reply by defunctzombie** [reply]
This will be fixed in the 2.7.0 release. Thanks for reporting.
